bitnami/phpbb/templates/deployment.yaml

```diff
@@ -1,166 +1,242 @@
+            - name: PHPBB_DATABASE_PORT_NUMBER
+              value: {{ .Values.externalDatabase.port | quote }}
+            - name: PHPBB_DATABASE_NAME
+              value: {{ .Values.externalDatabase.database | quote }}
+            - name: PHPBB_DATABASE_USER
+              value: {{ .Values.externalDatabase.user | quote }}
+            - name: PHPBB_DATABASE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ printf "%s-%s" .Release.Name "externaldb" }}
+                  key: db-password
+          {{- end }}
+            - name: PHPBB_SKIP_BOOTSTRAP
+              value: {{ default "no" .Values.phpbbSkipInstall | quote }}
+            - name: PHPBB_USERNAME
+              value: {{ default "" .Values.phpbbUsername | quote }}
+            - name: PHPBB_DISABLE_SESSION_VALIDATION
+              value: {{ default "no" .Values.phpbbDisableSessionValidation | quote }}
+            - name: PHPBB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "phpbb.secretName" . }}
+                  key: phpbb-password
+            - name: PHPBB_EMAIL
+              value: {{ default "" .Values.phpbbEmail | quote }}
+            - name: SMTP_HOST
+              value: {{ default "" .Values.smtpHost | quote }}
+            - name: SMTP_PORT
+              value: {{ default "" .Values.smtpPort | quote }}
+            - name: SMTP_USER
+              value: {{ default "" .Values.smtpUser | quote }}
+            - name: SMTP_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "phpbb.secretName" . }}
+                  key: smtp-password
+            - name: SMTP_PROTOCOL
+              value: {{ default "" .Values.smtpProtocol | quote }}
+            {{- if .Values.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+          {{- if .Values.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.containerPorts.http }}
+            - name: https
+              containerPort: {{ .Values.containerPorts.https }}
+          {{- if .Values.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: /app.php/help/faq
```

Comment:
Could you confirm that the FAQ will try and connect to the database?

Reply:
It fails if there's no database